rust_src/src/lisp.rs

```diff
@@ -246,118 +462,182 @@ macro_rules! defun {
 /// of arguments.
 pub const MANY: i16 = -2;
 
-/// Convert a LispObject to an EmacsInt.
-#[allow(non_snake_case)]
-fn XLI(o: LispObject) -> EmacsInt {
-    o as EmacsInt
-}
+mod deprecated {
```

Comment:
We should add a comment here to clarify what we mean by deprecated. I think we mean functions that we've written to make porting easy, but we should be able to remove once the relevant functionality is Rust-only. Is that correct?